src/components/Intro.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import styled from 'styled-components';
+import { OuterWrapper, InnerWrapper } from './reusable/Wrappers';
+
+export const Intro = () => {
+  const IntroSection = styled.section`
+    display: flex;
+    justify-content: center;  
+    background: white;
+  `;
+  const IntroText = styled.div`
```

Comment:
In my portfolio I have tried to put as much as possible in a global component to make the styled components more reusable. I have added styling in an existing styled component like this example bellow:

export const Maintext = styled(Techtext)
    text-align: center;


**IF** you want the same size and font-style of your <p> but only change things like if it's centered or the color, I believe this is a good way to do it.